contrib/lib/src/space_helmet/helmet.rs

```diff
@@ -0,0 +1,131 @@
+use std::sync::atomic::{AtomicBool, Ordering};
+
+extern crate rocket;
```

Comment:
We shouldn't need this `extern crate` here since `contrib` already has `extern crate rocket`.